### PR TITLE
Linear layout 2.0

### LINKED ISSUE #1257: 2.0 - Update LayoutGrid
This is a tracking issue for updating LayoutGrid

**Scope:** LayoutGrid
**Docs:** https://material.io/develop/web/components/layout-grid/

### PR DESCRIPTION
**Pull request by cromefire**
Fixes #1257

Linear layout is not yet available in 1.X or 2.X

### CODE REVIEW

**Comment by cromefire** [comment]
@prateekbh where is that "login" page in applitools coming from?

File: packages/layout-grid/package.json [hunk 1]
```diff
@@ -17,6 +17,8 @@
   "license": "MIT",
   "sass": "sass",
   "main": "lib/index.js",
+  "types": "lib/index.d.ts",
```
Comment (prateekbh) [inline packages/layout-grid/package.json]: ohhh we neeed to do this for every other package as well

Comment (cromefire) [reply packages/layout-grid/package.json]: Yes will do Search+Replace

**Comment by cromefire** [comment]
Note that I want to leave this open until the base packages has been updated

**Comment by prateekbh** [comment]
@cromefire re: Login screen: I was just messing around some stuff...

**Comment by prateekbh** [comment]
oh I thought you wanna do that as a separate PR?

File: packages/layout-grid/package.json (hunk 1 above)
Comment (cromefire) [reply packages/layout-grid/package.json]: Part of #1325

**Comment by prateekbh** [comment]
@cromefire I dont think the upgrade to base packages is coming anytime soon. should we just ship this with 0.41?

**Comment by cromefire** [comment]
If you think that's a good idea. I didn't see any deprecation notice so I don't know, if there is something happening.

**Comment by prateekbh** [comment]
I guess it should be fine

**Pull request merged**